coreneuron/nrniv/main1.cpp

```diff
@@ -453,32 +470,33 @@ extern "C" int run_solve_core(int argc, char** argv) {
         }
         // Set the buffer size if is not the default value. Otherwise use report.conf on
         // register_report
-        if (!nrnopt_is_default_value("--report-buffer-size")) {
+        if (cn_par.report_buff_size!=4) {
```

Comment:
I think that testing if the `report_buff_size` is equal to the number `4` is not a very good idea, since the default value may change in the future. If there is not any better way to check if an option is given in `CLI11` you could use an extra member with the default `buff_size` and check the `cn_par.report_buff_size` with it at least

Reply:
Probably not the smartest idea indeed. Thanks for your advice, I'll find a better solution to this.